updates/23/asset_collections_mappings_updates.sql

```diff
@@ -5,4 +5,5 @@ INSERT INTO multiasset_mappings(collection_id, asset) VALUES (261, "STRK");
 INSERT INTO multiasset_mappings(collection_id, asset) VALUES (217, "eip155:100/erc20:0x6C76971f98945AE98dD7d4DFcA8711ebea946eA6");
 *
 INSERT INTO multiasset_mappings(collection_id, asset) VALUES (248, "eip155:42161/erc20:0x1509706a6c66CA549ff0cB464de88231DDBe213B");
-*
\ No newline at end of file
+*
+DELETE FROM multiasset_mappings WHERE collection_id=155 AND asset="eip155:137/erc20:0x24834BBEc7E39ef42f4a75EAF8E5B6486d3F0e57";
```

Comment:
This deletes wrapped terra luna from the terra luna group as the wrapped one seems to have a completely different price. Perhaps rebasing?